oracledb-ballerina/types.bal

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+import ballerina/sql;
+
+
+# IntervalYearToMonth stores a period of time in years and months.
```

Comment:
```suggestion
# Stores a period of time in years and months.
```

Reply:
Check all the places. We don't need to duplicate record name here as it renders properly under record name in the generated API docs

Reply:
Fixed with a2696e1433d2aa6528411382dd4340e4357a845c